pwa/src/components/dashboardCard/DashboardCard.tsx

```diff
@@ -10,33 +10,59 @@ import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { dateTime } from "../../services/dateTime";
 import { faArrowRight, faTrash } from "@fortawesome/free-solid-svg-icons";
 import { getStatusTag } from "../../services/getStatusTag";
+import { Button } from "../button/Button";
 
 export interface DashboardCardProps {
   title: {
     label: string;
     href: string;
   };
   type: string;
-  status?: string | boolean;
-  isEnabled?: boolean | undefined;
-  lastRun?: string;
-  lastCall?: string;
-  onDelete?: (e: React.MouseEvent<HTMLButtonElement, MouseEvent> | React.TouchEvent<HTMLButtonElement>) => void;
-}
+  tags: {
+    label: string;
+    tooltip: string;
+  }[];
+  onDelete: (e: React.MouseEvent<HTMLButtonElement, MouseEvent> | React.TouchEvent<HTMLButtonElement>) => void;
 
-export const DashboardCard: React.FC<DashboardCardProps> = ({
-  title,
-  type,
-  status,
-  isEnabled,
-  lastRun,
-  lastCall,
-  onDelete,
-}) => {
-  const { t, i18n } = useTranslation();
+  isEnabled?: boolean;
+}
 
+export const DashboardCard: React.FC<DashboardCardProps> = ({ title, type, tags, isEnabled, onDelete }) => {
   return (
-    <div className={styles.container} onClick={() => navigate(title.href)}>
+    <div className={styles.container}>
+      <div className={styles.headerContainer}>
+        <div>
+          <span>{title.label}</span>
+
+          {isEnabled !== undefined && (
+            <ToolTip tooltip="Is enabled">
+              <span className={clsx(styles.enabledIndicator, isEnabled && styles.enabled)} />
+            </ToolTip>
+          )}
+        </div>
+        <span className={styles.cardType}>{type}</span>
+      </div>
+
+      <ul className={styles.tagsList}>
+        {tags.map((tag, idx) => (
+          <li key={idx}>
+            <ToolTip tooltip={tag.tooltip}>
+              <Tag label={tag.label} />
+            </ToolTip>
+          </li>
+        ))}
+      </ul>
```

Comment:
Right now, when there are no `tags` available (which can definitely happen!), we still render the `<ul>` element (and its styling), creating an unintended layout:

![image](https://user-images.githubusercontent.com/66552932/220939340-9d18cd64-3b17-4e34-bf40-4776ba5eae10.png)

Effectively adding `26px`.

We should first check if the `tags[]` has items, before rendering the `<ul/>`, like so (dummy code):

```
{!!tags.length && (<ul> ... </ul>)}
```